signac/synced_collections/numpy_utils.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) 2020 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+"""Define utilities for handling NumPy arrays."""
+
+
+class NumpyConversionWarning(UserWarning):
+    """Warning that NumPy data was converted."""
```

Comment:
I don't think the implicitly is necessary since the whole point of the warning is to be more open about it. Technically it's probably fine language to use, but I don't think it adds anything to the warning. The rest of the rewording is good though.